src/libs/numberFormat/index.native.js

```diff
@@ -0,0 +1,14 @@
+// we only need polyfills for Mobile.
+import '@formatjs/intl-getcanonicallocales/polyfill';
+import '@formatjs/intl-locale/polyfill';
+import '@formatjs/intl-pluralrules/polyfill';
+import '@formatjs/intl-numberformat/polyfill';
+
+// Load en Locale data
+import '@formatjs/intl-numberformat/locale-data/en';
```

Comment:
This part is important. It will enable a particular locale. So if we want to support more we have to load its respective file. A proper way would be when the user switches a locale, load the file remotely.

Reply:
This is fine for now